dev/com.ibm.ws.security.spnego_fat.2/fat/src/com/ibm/ws/security/spnego/fat/FATSuite.java

```diff
@@ -0,0 +1,140 @@
+/*******************************************************************************
+ * Copyright (c) 2014, 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.security.spnego.fat;
+
+import java.io.IOException;
+import java.net.InetAddress;
+
+import org.junit.ClassRule;
+import org.junit.rules.ExternalResource;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.junit.runners.Suite.SuiteClasses;
+
+import com.ibm.websphere.simplicity.log.Log;
+import com.ibm.ws.security.spnego.fat.config.CommonTest;
+import com.ibm.ws.security.spnego.fat.config.InitClass;
+import com.ibm.ws.security.spnego.fat.config.SPNEGOConstants;
+
+import componenttest.topology.impl.JavaInfo;
+import componenttest.topology.impl.JavaInfo.Vendor;
+import componenttest.topology.impl.LibertyServerFactory;
+
+@RunWith(Suite.class)
```

Comment:
No repeats for Jakarta 9?  I saw the bnd file said it was going to test faces-3.0

Reply:
The FATs were originally all in one bucket, and took ~1.5 hours to run end to end in FULL mode (and overran the trace limit by quite a bit), and I didn't want to add another 1.5 hours. So I spoke with @utle and we picked a few of the FATs that represented a good coverage and then split the bucket into 3 buckets to get them each around 30 mins and below the trace output limit for the bucket.

Reply:
(I think this is the bucket that didn't have one FATs we picked for EE9 repeat)

Reply:
The tested features is probably b/c I got them all running first and then split the buckets by copying from the original bucket. I can remove that tested.features.